hooks/blueprint

```diff
@@ -283,7 +283,9 @@ if [[ "${version}" != "" ]] && ! new_enough "${version}" "1.15.2"; then
 fi
 # this is only used for upgrading from 1.15.2 to 2.0 and can be removed in later versions
 if [[ "${version}" != "" ]] && ! new_enough "${version}" "2.0" ; then
- merge+=( overlay/addons/change-db-owner.yml )
+ if ! want_feature "external-db-mysql" ; then
+  merge+=( overlay/addons/change-db-owner.yml )
+ fi
 fi
```

Comment:
No sense nesting ifs...

```
if [[ "${version}" != "" ]] && ! new_enough "${version}" "2.0"  && ! want_feature "external-db-mysql" ; then
  merge+=( overlay/addons/change-db-owner.yml )
fi
```

Reply:
updated with no nested ifs and using "+external-db"